src/CustomJWT.h

```diff
@@ -157,7 +157,7 @@ class CustomJWT
               size_t outLen,
               char *alg,
```

Comment:
Any reason this part in the constructor was not made const?

Reply:
I'm so sorry, please check again. thank's